.github/workflows/ci.yaml

```diff
@@ -32,6 +32,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.sha }}
+          fetch-depth: 0
+
+      - name: Rebase
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git fetch origin ${{ github.base_ref }}
+          git rebase origin/${{ github.base_ref }}
```

Comment:
this is unrelated to this PR but this code does not work for user that are outside for the ElementsProject

If I merge a PR the Ci on master just fails with

```
Run git config user.name github-actions
  git config user.name github-actions
  git config user.email github-actions@github.com
  git fetch origin 
  git rebase origin/
  shell: /usr/bin/bash -e {0}
  env:
    RUST_PROFILE: release
    SLOW_MACHINE: 1
    RUST: 1
    COMPAT: 1
    BOLTDIR: bolts
  
fatal: invalid upstream 'origin/'
Error: Process completed with exit code 128.
```

Reply:
Huh. There's probably a GH variable we should use here?

Reply:
I have no idea, but from the following blog post https://www.jessesquires.com/blog/2021/10/17/github-actions-workflows-for-automatic-rebasing-and-merging/ I see that maybe there is no need to specify the `origin/`?

overall in the CI we should have a single remote